test/src/org/omegat/core/data/ExternalTMFactoryTest.java

```diff
@@ -87,7 +85,7 @@ public void testLoadTMX() throws Exception {
 
         assertTrue(ExternalTMFactory.isSupported(tmxFile));
 
-        ExternalTMX tmx = ExternalTMFactory.load(tmxFile);
+        ExternalTMX tmx = ExternalTMFactory.load(tmxFile, props, segmenter, filterMaster);
```

Comment:
The order of argument.
`ExternalTMFactory.load` takes `segmenter` then `filterMaster` but `AlignFilesCallback` ctor takes `filtermaster` then `segmenter`.